src/configuration/app/build.md

```diff
@@ -47,7 +47,7 @@ dependencies:
     behave: '*'
   ruby: # Specify one Bundler package per line.
     sass: '3.4.7'
-  nodejs: # Specify one NPM package per line.
+  nodejs:6.9 # Specify one NPM package per line.
     grunt-cli: '~0.1.13'
```

Comment:
Woah, what?  Since when can we specify a version here?  That requires more than a passing change.  Needs more info.